modulepurepy/interface.py

```diff
@@ -183,19 +183,8 @@ def update(self, **kwargs):
         if l is not None and u is not None:
             self._model.update_bounds(l, u)
 
-        if q is None and \
```

Comment:
Same issue as in the module function when wrong parameters are passed

Reply:
Done.